test/e2e/lib/gutenberg/gutenberg-editor-component.js

```diff
@@ -652,4 +653,19 @@ export default class GutenbergEditorComponent extends AsyncBaseContainer {
 			}
 		}
 	}
+
+	async clickUpgradeOnPremiumBlock() {
```

Comment:
@tjcafferkey, can we move this to [payment-block-component.js](https://github.com/Automattic/wp-calypso/blob/83257e742a31764a3540e35836d034cb44ed0c02/test%2Fe2e%2Flib%2Fgutenberg%2Fblocks%2Fpayment-block-component.js)?

Reply:
I think it would make more sense to move it there indeed! If you need anything from me to do so then I'm happy to help. 😄

Reply:
I guess it would be a good opportunity to do it along with the fix for the skipped spec in https://github.com/Automattic/wp-calypso/pull/48173